src/main/kotlin/solve/importer/view/DirectoryPathView.kt

```diff
@@ -19,55 +22,80 @@ class DirectoryPathView : View() {
 
     private val directoryChooser = DirectoryChooser().apply { title = "Choose working directory" }
 
-    private val filesCountIcon = loadResourcesImage(IconsImporterCheckCirclePath)
-    private val errorsCountIcon = loadResourcesImage(IconsImporterWarningPath)
+    private val roboto = Style.fontCondensed
 
-    private val directoryLabel = label("Project directory") {
-        maxWidth = 300.0
-        textOverrun = OverrunStyle.ELLIPSIS
-        controller.directoryPath.onChange {
-            tooltip(controller.directoryPath.value)
-            val x = controller.directoryPath.value
-            this.text = if (x == null) "Project directory"
-            else "Project directory\n$x"
+    private val directoryField = mfxTextField {
+        isEditable = false
+        isAllowEdit = false
+
+        enableWhen {
+            controller.projectAfterPartialParsing.isNotNull
         }
-    }
+        textFill = Color.valueOf(Style.onBackgroundColor)
 
-    private val changeButton = button("Change") {
-        action {
-            val dir = directoryChooser.showDialog(currentStage)
-            controller.directoryPath.set(dir?.absolutePath)
+        prefHeight=48.0
+        prefWidth=280.0
+        BorderPane.setAlignment(this, Pos.CENTER)
+        BorderPane.setMargin(this, Insets(16.0, 0.0,0.0, 0.0))
+
+        style = "-fx-border-color: #${Style.primaryColorLight}; -fx-font-size: 15px;"
+
+        floatMode = FloatMode.BORDER
+        font = Font.font(roboto, 14.0)
+
+        floatingText = "Select project directory"
+        controller.directoryPath.onChange {
+            floatingText = if (it != null && it != ""){
+                "Selected project directory"
```

Comment:
The text is overlapped 
![Ck6bRtri8b](https://user-images.githubusercontent.com/55746901/232252121-e05f9da8-6013-4150-9c5d-e448f1263ec7.gif)